src/Jupyter/Visualization/StateDisplayEncoders.cs

```diff
@@ -378,11 +390,51 @@ public StateVectorToHtmlResultEncoder(IConfigurationSource configurationSource)
                             </tr>
                         </thead>
 
+                        <tbody>
+                            {formattedData}
+                        </tbody>
+                    </table>
+                "; 
+                if (ConfigurationSource.PhaseDisplayStyle == PhaseDisplayStyle.None) {
+                    outputTable = $@"
+                    <table style=""table-layout: fixed; width: 100%"">
+                        <thead>
+                            {qubitIdsRow}
+                            <tr>
+                                <th style=""width: {basisWidth}ch)"">Basis state{basisStateMnemonic}</th>
+                                <th style=""width: 20ch"">Amplitude</th>
+                                <th style=""width: calc(100% - 26ch - {basisWidth}ch)"">Meas. Pr.</th>
+                            </tr>
+                        </thead>
+
+                        <tbody>
+                            {formattedData}
+                        </tbody>
+                    </table>
+                ";
+                };
+                if (ConfigurationSource.MeasurementDisplayStyle == MeasurementDisplayStyle.None) {
+                    outputTable = $@"
+                    <table style=""table-layout: fixed; width: 100%"">
+                        <thead>
+                            {qubitIdsRow}
+                            <tr>
+                                <th style=""width: {basisWidth}ch)"">Basis state{basisStateMnemonic}</th>
+                                <th style=""width: 20ch"">Amplitude</th>
+                                <th style=""width: 6ch"">Phase</th>
+                            </tr>
+                        </thead>
+
                         <tbody>
                             {formattedData}
                         </tbody>
                     </table>
                 ";
+                };
+                
+                if (ConfigurationSource.MeasurementDisplayHistogram == true) {
```

Comment:
Minor suggestion, but `== true` isn't needed when testing a `bool` value:
```suggestion
                if (ConfigurationSource.MeasurementDisplayHistogram) {
```